contrib/completions/bash/buildah

```diff
@@ -609,12 +615,15 @@ return 1
 
  _buildah_from() {
      local boolean_options="
+     --cert-dir
+     --creds
      --help
      -h
      --pull
      --pull-always
      --quiet
      -q
+     --tls-verify
```

Comment:
This _is_ a boolean option, but it duplicates a value listed in `$options_with_args`, which should be removed from that while we're adding this one here.

Reply:
touched this up as well, fixed ty.